packages/ag-charts-community/src/chart/series/cartesian/areaSeries.ts

```diff
@@ -596,7 +596,7 @@ export class AreaSeries extends CartesianSeries<
 
         const xString = xAxis.formatDatum(xValue);
         const yString = yAxis.formatDatum(yValue);
-        const title = sanitizeHtml(yName);
+        const title = yName != null ? sanitizeHtml(yName) : undefined;
```

Comment:
Nit: Just wondering if we should change `sanitizeHtml()` to return `undefined` if it's input is `undefined`, since it looks like the majority of uses are guarding against this - so it might be cleaner overall?

Reply:
You could probably change the signature of `sanitizeHtml()` using conditional return types to help flush out any issues with that change? E.g.:

```
export function sanitizeHtml<T extends string | undefined>(text: T): T extends undefined ? undefined : string {
```